include/deal.II/fe/mapping.h

```diff
@@ -962,16 +962,15 @@ class Mapping : public Subscriptor
    *   this object also represents with respect to which cell the transformation
    *   should be applied to.
    * @param[out] output An array (or part of an array) into which the transformed
-   *   objects should be placed.
+   *   objects should be placed. (Note that the array view is @p const, but the
+   *   tensors it points to are not.)
```

Comment:
A side remark is that while these interfaces are public they are kind of internal in the sense that almost no user will ever need to see them. Thus, the issue is probably not too important.